### PR TITLE
Template updates 

### PR DESCRIPTION
**Pull request by brandonhundt**
- Nav starter
- fonts
- bootstrapping layouts
- more responsive tabs

To Review:
- Do a functional review!

### CODE REVIEW

File: app/views/layouts/_nav.html.erb [hunk 1]
```diff
@@ -1,5 +1,55 @@
...
+        <ul class="dropdown-menu dropdown-menu-end shadow" aria-labelledby="navbarDropdown">
+          <li><a class="dropdown-item" href="https://help.prx.org/">Help Docs</a></li>
+          <li><a class="dropdown-item" href="http://status.prx.org/">Status</a></li>
+          <li><a class="dropdown-item" href="https://exchange.prx.org/terms-of-use">Terms</a></li>
+          <li><a class="dropdown-item" href="https://dovetail.prx.org/attribution">Attribution</a></li>
+          <li><a class="dropdown-item" href="https://exchange.prx.org/privacy-policy">Privacy</a></li>
+          <li><a class="dropdown-item" href="https://prx.org">PRX</a></li>
+        </ul>
+      </li>
+      <% if current_user %>
```
Comment (cavis) [inline app/views/layouts/_nav.html.erb:24]: Yeah, doesn't exist yet.  Will work on that soon.

We may also want to split this file out a bit at some point.  Getting a bit large.

File: app/views/layouts/_subnav.html.erb [hunk 2]
```diff
@@ -1,8 +1,38 @@
-<% if content_for? :subnav %>
-  <nav class="navbar navbar-dark bg-secondary">
-    <div class="container">
-      <%= link_to 'Subnav', fake_index_path, class: 'navbar-brand' %>
-      <%= yield :subnav %>
+<nav class="prx-app-nav navbar bg-white d-flex flex-row flex-wrap justify-content-start m-0 p-0">
+
+    <!-- PODCAST SWITCHER -->
```
Comment (cavis) [inline app/views/layouts/_subnav.html.erb:3]: Some more indent issues - I'll install an html/erb linter soon as well.  So this is fine!

**Review by cavis — APPROVED** [review]
Some things to lint, some things to plan (shared styles).  But I think this is going in the right direction, and it's not real yet, so onwards!

File: app/views/layouts/_nav.html.erb (hunk 1 above)
Comment (brandonhundt) [reply app/views/layouts/_nav.html.erb:24]: +1 Maybe pull the account stuff into its own layout. For another day :)

File: app/assets/stylesheets/shared/header.scss [hunk 3]
```diff
@@ -1,88 +1,134 @@
...
-  header {
-    border-bottom: 1px solid #e6e6e6;
-    background: $gray-900;
-    padding: 0;
-    z-index: 9999;
+ .navbar-brand {
+  display: block;
+  width: 152px;
+  height: 30px;
+  background-image: image-url("dovetail_logo.svg");
```
[the remark this answers is not in the document]
Comment (brandonhundt) [reply app/assets/stylesheets/shared/header.scss:8]: +1 to shared styles, I don't think they matter until we update the nav in ID and Augury.

**Pull request merged**